config/vufind/EDS.ini

```diff
@@ -212,3 +212,18 @@ organization_id = "VuFind 2.x from MyUniversity"
 [List]
 view=full
 
+; This section controls the behavior of the Autocomplete of EDS 
+; If enabled the option "autocomplete" is send for UIDAuth to get the token
+; and the url. 
+[Autocomplete]
+; Set this to false to disable all autocomplete behavior
+enabled = true
+
+; Define a default_handler 
+default_handler = Eds
+
+; Indicates the domain/data type to autocomplete
+; Use holdings for title completion in PubFinder. 
+; Use rawqueries for completion of basic textual queries. 
+idx = rawqueries
```

Comment:
Instead of configuring this as a setting here, would it make more sense to make this the configuration of the autocomplete handler? Then you could say `Eds:rawqueries` or `Eds:holdings` in association with different search handlers if it made sense to use different settings in different contexts. Since you are not currently using the setConfig method of your Eds handler, shifting things around a little bit might make everything more flexible.

Reply:
I'm not sure, if your approach make the configuration clearer. The idx parameter is part of the documentation in the eds-wiki. So the recognition is higher if we use the same parameter name.
That is my opinion, but I can change it, if you prefer.

Reply:
I think it depends on the use case. If a user is only ever going to want to perform one type of searching, having the separate setting here is more clear. However, if a user might want to do 'rawqueries' searching for one kind of search and 'holdings' for a different type, using the configuration capability of the autocomplete handler allows flexibility, so you could do something like:

<pre>
[Autocomplete_Types]
AllFields = Eds:rawqueries
TI = Eds:holdings
</pre>
</pre>